Build/BUILDsis2

```diff
@@ -33,6 +33,15 @@ set -x
   for arg in "$@"
   do
       case $arg in
+        prod|repro|debug)
+          comp="${arg#*=}"
+          if [ ${arg#} = 'repro' ] ; then
+            COMP="REPRO=Y"
```

Comment:
I think you should initialize COMP=""around line 31

Reply:
done